server/src/main/java/org/elasticsearch/action/admin/cluster/node/hotthreads/TransportNodesHotThreadsAction.java

```diff
@@ -32,21 +34,17 @@ public class TransportNodesHotThreadsAction extends TransportNodesAction<
     TransportNodesHotThreadsAction.NodeRequest,
     NodeHotThreads> {
 
+    private static final String NAME = "cluster:monitor/nodes/hot_threads";
+    public static final ActionType<NodesHotThreadsResponse> TYPE = new ActionType<>(NAME, Writeable.Reader.localOnly());
```

Comment:
Could we encapsulate this as `ActionType.localOnly(NAME)`? There's already a fair few of them and lots more on the way, see https://github.com/elastic/elasticsearch/issues/100111.

Reply:
++ added